button.md

```diff
@@ -0,0 +1,85 @@
+## Quick Start
+
```

Comment:
You got it! I think I’m going to replace the “Quick Start” headers, since that doesn’t actually mean anything.